docs/modules/cp-subsystem/pages/cp-subsystem.adoc

```diff
@@ -118,6 +118,21 @@ For example, a cluster has three CP members and three CP groups, each CP member
 Raft leader for only one CP group. If one more CP group is created, then one of
 the CP members becomes the Raft leader for two CP groups.
 
```

Comment:
```suggestion

You can also give CP members a priority rating to affect their chances of becoming a leader. See xref:configuration.adoc#configuring-leadership-priority[Configuring Leadership Priority].
```

This is just a nudge to the reader that it's possible to do this task. The actual information is about configuration and belongs in the Configuring CP Subsystem topic.